### DISCUSSION TITLE
Structures in mobile view

### DISCUSSION THREAD

**Discussion by thupsi** [craftcms/cms#16190] [Ideas] [answered]
In Craft 5 the `table` view has been ditched in favour of the new `card` view for element indexes in mobile view. 

While this is a good UI for `channel` and `single`  sections, it falls apart when used for `structure` sections. 

Example of a structure in desktop mode and the same structure in mobile:

![image](https://github.com/user-attachments/assets/3fd7b03b-3280-402a-86ee-cf323ca8f6d9)

There is no way to infer the entries' hierarchy, nor a way to sort them in a predictable manner according to that hierarchy.

Some solutions that come to mind:
- Bring back the table view for structures in mobile view.
- Introduce levels of indentation for cards in mobile view (for structures). That would probably require the layout to allow for some horizontal scrolling, depending on the level of indentation. It would also not solve the problem of not being able to collapse/expand nodes in the structure.
- Sort the elements by "structure" order, and display the parent(s), breadcrumb like, on each card, maybe above the title.

**Comment by tomkiss** [comment]
Just had mild panic that this was a configuration option gone rogue.

Just adding a +1, seems like a pretty bad UI bug.

**Comment by brandonkelly** [comment, marked as answer]
Good point. Just brought structure view back (sans additional table columns) for Craft 5.6 (9fc09e72bc0bd218c3d04c0e40bf49e9a06938ca).

**Reply by brandonkelly** [reply]
Craft 5.6.0 is out with that change 🎉
